edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/Microsoft.Azure.Devices.Edge.Agent.Core.csproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\dotnetVersion.props" />
```

Comment:
please use netstandard2.0 for libraries

Reply:
From Andrew:
> I tried to use netstandard, and found this issue which instructs us to target netcoreapp directly:
https://github.com/dotnet/sdk/issues/3667

Reply:
this should be valid for projects that use asp.net, but is it not working for core project to use netstandard?

Reply:
I made this change to netstandard 2.1, but it causes a problem building. I investigated for a while then started trying to find a workaround. I've altered the buildBranch script so that we don't ```dotnet build``` the tests to an output folder, but rather ```dotnet publish``` them to an output folder. I don't know why we weren't doing this initially.

There are still issues running the tests on linux. Still working on fixing them. Windows is working fine.

Reply:
I figured out how to fix the build with netstandard without needing to switch build to publish.